mmv1/products/cloudidentity/Group.yaml

```diff
@@ -50,8 +51,18 @@ examples:
     test_env_vars:
       org_domain: :ORG_DOMAIN
       cust_id: :CUST_ID
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'cloud_identity_groups_dynamic'
+    skip_test: true
+    primary_resource_id: 'cloud_identity_groups_dynamic'
+    vars:
+      id_group: 'my-identity-dynamic-group'
+    test_env_vars:
+      org_domain: :ORG_DOMAIN
+      cust_id: :CUST_ID
 custom_code: !ruby/object:Provider::Terraform::CustomCode
-  post_create: templates/terraform/post_create/set_computed_name.erb
+  pre_create: templates/terraform/pre_create/hide_dynamic_label.erb
```

Comment:
Noting for myself: This resource's labels were excluded from the provider-wide labels rework as they were not a standard labels field. Values could only be set to `""`.